### PR TITLE
Add a PostGIS layer to QGIS

### PR DESCRIPTION
**Pull request by pvalsecc**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -23,3 +23,19 @@ VISIBLE_WEB_HOST ?= geomapfish-demo.camptocamp.com
...
+	docker run -ti \
+	  --add-host=db:$$IP \
+	  --volume $$PWD/qgisserver:/project \
+	  --volume /tmp/.X11-unix:/tmp/.X11-unix \
+	  --volume $$HOME:$$HOME \
+	  --env DISPLAY=unix$${DISPLAY} \
+	  --rm camptocamp/qgis-server:latest /usr/local/bin/start-client.sh
+
+qgisserver/project.qgs.tmpl: qgisserver/project.qgs Makefile
+	sed -e "s/user='[^']*'/user='\$${PGUSER}'/g; s/password='[^']*'/password='\$${PGPASSWORD}'/g; s/host=db/host=\$${PGHOST}/g; s/port=[0-9]*/port=\$${PGPORT}/g; s/dbname='[^']*'/dbname='\$${PGDATABASE}'/g" < $< > $@
```
Comment (sbrunner) [inline Makefile]: With that we can't have more than one database, right?

Comment (pvalsecc) [reply Makefile]: Exact.

Comment (sbrunner) [reply Makefile]: And what about doing one repacement to touch only the database host db?
I can do it in an other pull request if needed...

Comment (pvalsecc) [reply Makefile]: Good idea, changing that.

**Pull request merged**